schemas/lib/index-constructor-input.js

```diff
@@ -0,0 +1,41 @@
+import * as path from "path";
+
+import Joi from "joi";
+
+const cwd = process.cwd();
+
+export default Joi.object().keys({
```

Comment:
Can you do:
```javascript
export const interlockConstructor = Joi.object().keys({ // ...
```
And rename file to `schema/lib/index.js`, to avoid arbitrary filenames.